src/args/make-node-environment/default.nix

```diff
@@ -68,7 +68,7 @@ let
   };
 in
 makeSearchPaths {
-  envPaths = [ nodeEnvironment ];
+  envPaths = [ nodeInterpreter ];
```

Comment:
@kamadorueda Looks like commitlint needs node available to be able to run. What do you think about this approach?

Reply:
I think we can add node to the makeNodeEnvironment, 

makePythonEnvironment includes python